src/routes/+page.svelte

```diff
@@ -357,16 +349,116 @@
+        >
+            <MegaEth />
+
+            <div
+                class="absolute bottom-4 right-4 group-hover:inline-flex opacity-0 group-hover:opacity-100 transition-opacity duration-300"
+            >
+                <Visit />
+            </div>
+        </div>
+        <div
+            class="relative group cursor-pointer hover:bg-gray-lightest w-[296px] h-[296px] inline-flex justify-center items-center border"
+        >
+            <img src="/Vitalik.webp" alt="" width="296" height="296" />
+
+            <div
+                class="absolute bottom-4 right-4 group-hover:inline-flex opacity-0 group-hover:opacity-100 transition-opacity duration-300"
+            >
+                <Visit />
+            </div>
+        </div>
+        <div
+            class="relative group cursor-pointer hover:bg-gray-lightest w-[296px] h-[296px] inline-flex justify-center items-center border"
+        >
+            <EigenLayer />
+
+            <div
+                class="absolute bottom-4 right-4 group-hover:inline-flex opacity-0 group-hover:opacity-100 transition-opacity duration-300"
+            >
+                <Visit />
+            </div>
+        </div>
+        <div
+            class="relative group cursor-pointer hover:bg-gray-lightest w-[296px] h-[296px] inline-flex justify-center items-center border"
+        >
+            <EigenLayer />
+
+            <div
+                class="absolute bottom-4 right-4 group-hover:inline-flex opacity-0 group-hover:opacity-100 transition-opacity duration-300"
+            >
+                <Visit />
+            </div>
+        </div>
+        <div
+            class="relative group cursor-pointer hover:bg-gray-lightest w-[296px] h-[296px] inline-flex justify-center items-center border"
+        >
+            <EigenLayer />
+
+            <div
+                class="absolute bottom-4 right-4 group-hover:inline-flex opacity-0 group-hover:opacity-100 transition-opacity duration-300"
+            >
+                <Visit />
+            </div>
+        </div>
+        <div
+            class="relative group cursor-pointer hover:bg-gray-lightest w-[296px] h-[296px] inline-flex justify-center items-center border"
+        >
+            <EigenLayer />
+
+            <div
+                class="absolute bottom-4 right-4 group-hover:inline-flex opacity-0 group-hover:opacity-100 transition-opacity duration-300"
+            >
+                <Visit />
+            </div>
+        </div>
+        <div
+            class="relative group cursor-pointer hover:bg-gray-lightest w-[296px] h-[296px] inline-flex justify-center items-center border"
+        >
+            <EigenLayer />
+
+            <div
+                class="absolute bottom-4 right-4 group-hover:inline-flex opacity-0 group-hover:opacity-100 transition-opacity duration-300"
+            >
+                <Visit />
+            </div>
+        </div>
+    </div>
```

Comment:
Always keep the DRY (Don't Repeat Yourself) principle in mind. This code can be almost 100 lines shorter if you define an array of components and render them in a for loop


Also, between 1024 and 1311px of width, the items on the list start to overlap. Noticed that because I've open it using Arc's preview feature and the page looked like that

<img width="1157" alt="image" src="https://github.com/user-attachments/assets/7350acc1-e2f8-4947-9be0-c7bf7f6f8f65">